src/@chakra-ui/gatsby-plugin/components/Tag.ts

```diff
@@ -0,0 +1,44 @@
+import type { ComponentStyleConfig } from "@chakra-ui/theme"
+
+export const Tag: ComponentStyleConfig = {
+  parts: ["container", "label"],
+  baseStyle: {
+    container: {
+      display: "flex",
+      alignItems: "center",
+      bgImage:
+        "radial-gradient(46.28% 66.31% at 66.95% 58.35%,rgba(127, 127, 213, 0.2) 0%,rgba(134, 168, 231, 0.2) 50%,rgba(145, 234, 228, 0.2) 100%)",
+      marginBottom: "0.5em",
```

Comment:
Lets use the Chakra scale if possible.
```suggestion
      mb: 2,
```